grafana/README.md

```diff
@@ -0,0 +1,22 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster.
+* single: dashboards for a single resource(e.g. `node` or `pod`).
+
+**Notice**
+* summary dashboards use `alias` as the `cluster` template variable.
```

Comment:
No I'm just wondering where this wondrous `alias` label is coming from. I've never seen it before, so I thought that it is something that you have in your Prometheus configuration like an external label or something. Because it's not part of the labels exposed by those metrics.